internal/release/RELEASE-NOTES

```diff
@@ -1,20 +1,92 @@
-# 1Password Go SDK v0.2.1
+# 1Password Go SDK v0.3.0
 
 ## NEW
 
-- **`CreatedAt` and `UpdatedAt` item metadata:** Items and item overviews now expose attributes with their creation and last edit times.
-- **Resolving secrets in bulk**: With the `client.Secrets().ResolveAll` function, the SDK is now able to resolve multiple secrets at once, improving the performance of the operation.
+- **Support for item states**: You can now fetch an item's state using the SDK. `Item` and `ItemOverview` expose two states: `Active` and`Archived`.
+    - `Active`: An item located inside a vault. (Default)
+    - `Archived`: An item that has been moved to the Archive. 1Password doesn't include archived items in search results or suggest them when you fill in apps and browsers. You can keep archived items as long as you'd like.
+- **Filtering the return of item listing by state**: You can now filter the listed items by item state. The current exposed filter is `ByState`, which allows devs to filter `ItemOverviews` by their `State` .
 
 ## IMPROVED
 
-- **Support for new field types:** Items with `Address` and `Date` fields can now be created, retrieved, and edited using the 1Password SDK.
-- **Item sharing for attachments and documents**: Items with files attached now can also be shared using the `client.Items().Shares()` functions.
-- **Adding custom fields in sections automatically**: The SDK now automatically adds custom fields without a section to an empty section within the item, creating it if necessary.
-- **`Tags` in item overviews**: The return type of `Items().ListAll` now also contains the item tags.
-- **Broader item editing capabilities**: You are now able to use the `items.put` function on more items, including those with fields that are not directly editable through the SDK (such as legacy fields, passkeys etc.)
+- **Improved Deleting Capabilities:** You can now delete `Archived` items in the SDKs.
```

Comment:
maybe even under FIXED?

Consider rephrasing to:
 ```
**Deleting Archived Items:** The SDK now supports deleting items from the archive.
```

As "improved deleting capabilities" loses some nuance:
* it does not reference items
* "improved" is redundant with the title of the section